src/Features/LanguageServer/ProtocolUnitTests/SemanticTokens/SemanticTokensEditsTests.cs

```diff
@@ -114,14 +114,17 @@ public async Task TestGetSemanticTokensEdits_ReturnMinimalEdits()
 
             var results = await RunGetSemanticTokensEditsAsync(testLspServer, caretLocation, previousResultId: "1");
 
-            // 1. Replaces length of token (10 to 5) and replaces token type (comment to keyword)
+            // 1. Updates length of token (10 to 5) and updates token type (comment to keyword)
             // 2. Creates new token for '// Comment'
-            var expectedEdit = SemanticTokensEditsHandler.GenerateEdit(
-                start: 0, deleteCount: 5,
+            var expectedEdit = GenerateEdit(
+                start: 2, deleteCount: 0,
                 data: new int[]
                 {
-                    0, 0, 5, SemanticTokensCache.TokenTypeToIndex[LSP.SemanticTokenTypes.Keyword], 0,
-                    1, 0, 10, SemanticTokensCache.TokenTypeToIndex[LSP.SemanticTokenTypes.Comment], 0
+                    // 'class'
+                    /* 0, 0, */ 5, SemanticTokensCache.TokenTypeToIndex[LSP.SemanticTokenTypes.Keyword], 0,
```

Comment:
Ah, so this was actually intentional. Since we're switching over to computing edits by int now instead of by token, I thought it might be helpful to show where the inserted individual ints are located in relation to other ints. So for example, in the test above we can see that the new four new inserted ints make up the end of the `class` token and the beginning of the `// Comment` token. If you think this makes things more confusing, I don't mind removing it

Reply:
Ah I see, I think as-is is fine!